set.mm

```diff
@@ -21848,10 +21848,22 @@ proposition with a distinct variable (closed form of ~ nfsb4 ).
   $}
 
   ${
+    $d x y $.  $d y ph $.  $d y ps $.
     moimi.1 $e |- ( ph -> ps ) $.
     $( The at-most-one quantifier reverses implication.  (Contributed by NM,
-       15-Feb-2006.) $)
+       15-Feb-2006.)  Remove use of ~ ax-5 .  (Revised by Steven Nguyen,
+       9-May-2023.) $)
     moimi $p |- ( E* x ps -> E* x ph ) $=
+      ( vy weq wi wal wex wmo imim1i alimi eximi df-mo 3imtr4i ) BCEFZGZCHZEIAP
+      GZCHZEIBCJACJRTEQSCABPDKLMBCENACENO $.
```

Comment:
I think the change is good, but this is an instance where this is not too important since ~mojust already uses ax-1--ax-7.